### PR TITLE
SD Hunting Data

### PR DESCRIPTION
**Pull request by FishheadNate**
- Created Directory for SD
- Created json files for
  - species season units
  - walk-in areas
  - state & federal areas

### CODE REVIEW

File: sources/US/SD/school-land.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "name" : "SD School Lands",
```
Comment (JesseCrocker) [inline sources/US/SD/school-land.json]: Is this needed? If all state school trust lands are open for hunting, and this data is already in the public land layer then i dont think we need it, but i dont know if either of those things is true.

File: sources/US/SD/wpa-federal-lands.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "name" : "SD Waterfowl Production Areas & Federal Lands Open for Hunting",
```
Comment (JesseCrocker) [inline sources/US/SD/wpa-federal-lands.json]: Is this a duplicate of data that is already in the public land layer?

File: sources/US/SD/school-land.json (hunk 1 above)
Comment (FishheadNate) [reply sources/US/SD/school-land.json]: I didn't cross reference the Public Land layer. School land is already in there.

File: sources/US/SD/wpa-federal-lands.json (hunk 2 above)
Comment (FishheadNate) [reply sources/US/SD/wpa-federal-lands.json]: ~90% duplicate. Public Land layer is missing USACOE land along the Missouri River, will open an issue for that.

**Pull request merged**